app/assets/javascripts/discourse/tests/integration/components/bookmark-test.js

```diff
@@ -40,5 +40,9 @@ module("Integration | Component | bookmark", function (hooks) {
       "2020-05-15"
     );
     assert.strictEqual(query("#custom-time").value, "09:45");
+    assert.strictEqual(
+      query(".selected-name > .name").innerHTML.trim(),
+      "Keep bookmark"
```

Comment:
This would incorrectly be "Keep bookmark and clear reminder" before the fix.

Reply:
We also have a section for this in `spec/system/bookmarks_spec.rb` that you may like to add to, but this works just as well :) If you want to keep it this way I would recommend changing to use `I18n` to get the "keep bookmark" text

Reply:
Yes! Definitely should use the translation for this. 🙂